tests/src/core/resolve.js

```diff
@@ -21,34 +21,48 @@ describe('resolve', function () {
     const testContext = utils.testContext({ 'import/resolver': './foo-bar-resolver-v1' });
 
     expect(resolve( '../files/foo'
-      , Object.assign({}, testContext, { getFilename: function () { return utils.getFilename('foo.js'); } }),
+      , Object.assign({}, testContext, { getPhysicalFilename: function () { return utils.getFilename('foo.js'); } }),
```

Comment:
rather than modifying the existing tests, please leave these alone, so that the new `getPhysicalFilename` duplicated tests are clearly additions

Reply:
Okay, which do you prefer?

1. Duplicate all of the existing tests so that the only difference is `getFilename()` vs. `getPhysicalFilename()`
2. Revert the changes so that we're testing all of the use cases for `getFilename()` and not exercising `getPhysicalFilename()` except for the couple of tests demonstrating that `getPhysicalFilename()` is the preferred API and `getFIlename()` is a fallback. 
3. Same as 1, but add `getFilename() { throw new Error(...) }` to the mock context on all the new tests rather than have a separate test for the fallback.

Reply:
I was originally thinking option 1, but i really do like option 3.

Reply:
@ljharb These changes are done.